_posts/2025-08-26-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Using Strimzi Metrics Reporter to Expose Kafka Metrics in Prometheus Format"
+date: 2025-10-01
+author: Owen Corrigan
+---
+When deploying new Kafka cluster it may be one of your intentions to monitor the cluster using metrics.
+`Apache Kafka®` generates and exposes metrics that reflect how a system is operating. 
+These metrics are useful for monitoring, troubleshooting, tuning, and capacity planning when it comes to running your Kafka cluster.
+In essence, monitoring is crucial to ensure the health and performance of your Kafka clusters.
+`Prometheus` as a metrics monitoring solution has become increasingly common due to its popularity in the cloud-native ecosystem, and it is this format that we use to expose metrics.
+
+The [Strimzi Metrics Reporter](https://github.com/strimzi/metrics-reporter) project implements the Java `MetricsReporter` interface to expose metrics via `Prometheus` and it is available since Strimzi 0.48.0 release.
```

Comment:
"the Java `MetricsReporter` interface" -> "the Kafka `MetricsReporter` interface"

Reply:
Silly mistake. Changed